### ISSUE TITLE
Addressing Shaon's comments

### ISSUE THREAD

**Issue by miqmiq** [PhoenixBirdCreations/IPAM2021_ML#38]
Line 55: Replace "in an NSBH system" by "in systems with a NS".

Line 79: Remove "preliminaty"? I think Sushant commented something on this but I don't remember what.

Line 82: it seems that the label that is actually used is HasRemnant instead of HasRem. We could put a footnote here saying that even though the current label is indeed HasRemnant we use HasRem to make it short.

Line 103: add "systematic" after "statistical".

Line 106: replace "performance" by "speed".

Lines 124, 216, 224: replace "EM-bright". In lines 124 and 216I would say "probability of formation of tidally disrupted matter" as Shaon suggests. In line 224 I'd put "since a system can only have tidally disrupted matter when..."

Line 210: After this sentence (or before) I'd say that we cross-validate over the O2 dataset to obtain these hyperparameters. Or maybe we can just say this in the reply to Shaon. 

Line 311: say why we use these 3 EOS: BHF BBB2 and MS1 PP give the smallest and largest NS maximum masses (so we consider the extreme cases), respectively, and SLy is the most popular choice of EOS. 

Line 591: replace "HasSSM" by "other data products".

**Comment by miqmiq** [comment]
I've just realized that the line number don't correspond to the current version of the draft, since I used the commented pdf that Shaon shared.

**Comment by lmagana3** [comment]
@miqmiq: I implemented all but the following three changes from above: 
~Line 79: Remove "preliminaty"? I think Sushant commented something on this but I don't remember what.
~ Line 210: After this sentence (or before) I'd say that we cross-validate over the O2 dataset to obtain these hyperparameters. Or maybe we can just say this in the reply to Shaon.
~Line 311: say why we use these 3 EOS: BHF BBB2 and MS1 PP give the smallest and largest NS maximum masses (so we consider the extreme cases), respectively, and SLy is the most popular choice of EOS.

Regarding the EM bright, I changed the first appearance of the phrase so now it reads like, "In reality, the probability of the formation of tidally disrupted matter, i.e. an \ac{EM} bright event, is always smaller than ..." In this way, we don't have to change the wording elsewhere. However, if you prefer to change it everywhere then we can just easily edit that.
